css/stylesheet.css

```diff
@@ -107,4 +107,82 @@ h2 {
 .social a:hover {
     color: #adddef;
 }
+.aboutme-arrows {
```

Comment:
this is the same as .arrows above, if you need to make changes, add 2 classes rather than duplicate code

Reply:
resolved